lbrynet/wallet/ledger.py

```diff
@@ -57,9 +62,20 @@ def resolve(self, page, page_size, *uris):
             except URIParseError as err:
                 defer.returnValue({'error': err.message})
         resolutions = yield self.network.get_values_for_uris(self.headers.hash().decode(), *uris)
-        resolver = Resolver(self.headers.claim_trie_root, self.headers.height, self.transaction_class,
-                            hash160_to_address=lambda x: self.hash160_to_address(x), network=self.network)
-        defer.returnValue((yield resolver._handle_resolutions(resolutions, uris, page, page_size)))
+        defer.returnValue((yield self.resolver._handle_resolutions(resolutions, uris, page, page_size)))
```

Comment:
Does `return yield self.resolver.....` work? In twisted on py3 `defer.returnValue` is no longer necessary. I'm wondering if it could be dropped here.

Reply:
Good catch. `return yield` doesn't from my last experiments, but `return (yield d)` did. Gonna try that.